doc/ref_model/chapters/chapter03.md

```diff
@@ -298,6 +298,16 @@ It is expected that reference architectures will provide a service chain workflo
 
 There is also a need to provide specialised tools to aid troubleshooting of individual services and the communication between them in order to investigate issues in the performance of composed network functions. Minimally, there is a need to provide packet level and byte level counters and statistics as the packets pass through the service chain in order to ascertain any issues with forwarding and performance. Additionally, there is a need for mechanisms to trace the paths of selected subsets of traffic as they flow through the service chain.
 
+<a name="3.5.1.1"></a>
+#### 3.5.1.1 Introduction 
+Service function chaining (SFC) can be visualized as layered structure where Service function plane (SFC data plane ,consists of Service function forwarder, Classifier, Service function, Service function proxy) resides over Service function overlay network. At a very high level, a service function plane is a directed acyclic graph with the composing Servie functions being the vertices.  
+Service function chaining utilizes a service-specific overlay that creates the service topology.  The service overlay provides service function connectivity, built "on top" of the existing network topology. It leverages various overlay network technologies (e.g., Virtual eXtensible Local Area Network (VXLAN)) for interconnecting SFC data-plane elements and allows establishing Service Function Paths (SFPs).
+
+In Overlay network, packets are routed based on networking principles as use a suitable path for the packet from source to its destination in network. 
```

Comment:
Not sure what this means

"packets are routed based on networking principles as use a suitable path for the packet from source to its destination in network. "

Can you reword?

Reply:
@sureshkrishnan Hi, i see you have commented on old commit. Can you have look for recent changes as it has been modified from the commit you reviewed as content has been refined . Thanks .